device-types/Check Point/SG5800.yaml

```diff
@@ -0,0 +1,65 @@
+---
+manufacturer: Check Point
+model: SG5800
+slug: check-point-sg5800
+part_number: CPAP-SG5800-NGTP
+u_height: 1
+weight: 8.37
+weight_unit: kg
+is_full_depth: true
+comments: '[Check Point 5800 Data Sheet](https://www.checkpoint.com/downloads/products/5800-security-gateway-datasheet.pdf)'
+airflow: side-to-rear
+console-ports:
+  - name: ttyS0
+    label: CONSOLE
+    type: rj-45
+  - name: ttyS1
+    label: CONSOLE
+    type: usb-c
+power-ports:
```

Comment:
By the looks of the spec sheet these are replaceable PSUs, therefore they should be modeled as `module-bays` and then add the required PSUs as modules if they don't exist already